src/main/java/org/datavyu/plugins/mpv/MpvAwtMediaPlayer.java

```diff
@@ -42,7 +42,8 @@ private long getWindowID(Container container){
         if (container.getPeer() == null){
             throw new RuntimeException("Unable to retrieve window id");
         }
-        return ((WComponentPeer) container.getPeer()).getHWnd();
+//        return ((WComponentPeer) container.getPeer()).getHWnd();
```

Comment:
Why did this change? Error?

Reply:
I see. Then this is simple. We can't depend on it since it is not available for OSs that we would like to support.

If I understand correct MPV currently works with JavaFX and on Windows. Correct?

Reply:
Correct.